app/controllers/provider_interface/provider_agreements_controller.rb

```diff
@@ -0,0 +1,55 @@
+module ProviderInterface
+  class ProviderAgreementsController < ProviderInterfaceController
+    before_action :set_page_name
```

Comment:
`@page_name` seems always the same, so this can go into the view I think?

Reply:
yes